File: Multi_Language_CodeExamples/php/array_operations.php
<?php
$numbers = [1, 2, 3, 4, 5];

$doubled = array_map(fn($n) => $n * 2, $numbers);
echo "Doubled: " . implode(", ", $doubled) . "\n";

$sum = array_sum($numbers);
echo "Sum: " . $sum . "\n";
?>

File: Multi_Language_CodeExamples/perl/array_operations.pl
#!/usr/bin/perl

my @numbers = (1, 2, 3, 4, 5);

my @doubled = map { $_ * 2 } @numbers;
print "Doubled: @doubled\n";

my $sum = 0;
$sum += $_ for @numbers;
print "Sum: $sum\n";
